README.md

```diff
@@ -27,10 +27,71 @@
 1. [Sign up for a Rollbar account](https://rollbar.com/signup).
 2. For client-side Javascript, follow the [Browser Quick Start](https://docs.rollbar.com/docs/javascript#section-quick-start-browser) instructions. For Node.js, follow the [Server Quick Start](https://docs.rollbar.com/docs/javascript#section-quick-start-server).
 
+## Installation
+
+Using npm:
+```bash
+npm install --save rollbar
+```
+
+Using yarn:
+```bash
+yarn add rollbar
+```
+
+Using CDN:
+```html
+<script src="https://cdn.rollbar.com/rollbarjs/refs/tags/vVERSION/rollbar.min.js"></script>
```

Comment:
I'm not sure if this is the most idiomatic way of telling users they have to write the version where `vVERSION` is.